environment/vm/lima/deb/docker.go

```diff
@@ -30,8 +31,12 @@ func (d *Docker) PreInstall() error {
 
 // Install implements URISource.
 func (d *Docker) Install() error {
+	args := []string{"docker", "--channel", dockerChannel()}
+	if v := os.Getenv("DOCKER_INSTALL_VERSION"); v != "" {
+		args = append(args, "--version", v)
+	}
 	return d.Guest.Run("sh", "-c",
-		`curl -fsSL https://get.docker.com -o /tmp/get-docker.sh && sudo sh /tmp/get-docker.sh`,
+		`curl -fsSL https://get.docker.com | sh -s `+strings.Join(args, " "),
```

Comment:
I think you can still pass the args to the written file i.e.
```
... sudo sh /tmp/get-docker.sh` + strings.Join(args, " ")
```

Reply:
I found it better to directly pipe to the shell so the script is executed immediately without being stored locally but if you prefer to keep writing the script before exec let me know.

Reply:
Updated